packages/polaris-viz/src/components/LineChart/Chart.tsx

```diff
@@ -243,9 +251,11 @@ export function Chart({
         max: data[longestSeriesIndex].data.length - 1,
       });
 
+      const {x, y} = getXYFromEventType(event);
```

Comment:
At some point, not necessarily now, I think it would be nice to consolidate some of the tooltip positioning logic. I'm mostly thinking of what's in this function, what's in `getAlteredPosition` and what's in the actual tooltip component and whether there's a way to simplify it all a bit